### PR TITLE
Favorite Movie Add and Edit

### PR DESCRIPTION
**Pull request by anthogr45**

### CODE REVIEW

File: client/src/components/MovieCard.jsx [hunk 1]
```diff
@@ -9,15 +9,28 @@ import CardContent from '@mui/material/CardContent';
...
-import { EDIT_MOVIE } from '../utils/mutations';
+import { useFavoriteMovies } from '../pages/favMovie';
 
 export default function MovieCard({movie}) {
-  const [editMovie] = useMutation(EDIT_MOVIE);
+  // const [editMovie] = useMutation(EDIT_MOVIE);
   // about movie image, please ref: https://developer.themoviedb.org/docs/image-basics
   const imageUrl = `https://image.tmdb.org/t/p/w500/${movie.poster_path}`;
+
+  const { isFavorite, addFavoriteMovie, removeFavoriteMovie } = useFavoriteMovies();
```
Comment (jizhang80) [inline client/src/components/MovieCard.jsx:23]: In your useFavoriteMovies, there is no "isFavorite" function.

File: client/src/pages/favMovie.jsx
Comment (jizhang80) [inline client/src/pages/favMovie.jsx:1]: you need a isFavorite() function here.

File: package.json
Comment (jizhang80) [inline package.json:1]: Please do not leave those conflict signs here.
and I don't think you should install "@material-ui/core": "^4.12.3", "@material-ui/icons": "^4.11.2", in root package.json file.

**Review by jizhang80 — CHANGES_REQUESTED** [review]
as the comment in the code, please check it then submit it again. We need really workable code.

**Comment by anthogr45** [comment]
Further testing

**Pull request closed without merging**